packages/manager/apps/ips/src/pages/listing/manageOrganisations/components/OrganisationsDatagrid/manageOrganisationsDataGrid.spec.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { describe, it, expect, vi } from 'vitest';
+import { render, waitFor } from '@testing-library/react';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import orgList from '../../../../../../mocks/organisations/get-organisations.json';
+import { ManageOrganisationsDatagrid } from './manageOrganisationsDataGrid.component';
+
+const queryClient = new QueryClient();
+
+/** MOCKS */
+const useGetOrgListMock = vi.hoisted(() =>
+  vi.fn(() => ({ orgList, isLoading: true, error: undefined })),
+);
+
+vi.mock('@/data/hooks/organisation', () => ({
+  useGetOrganisationsList: useGetOrgListMock,
+}));
+
+/** RENDER */
+const renderComponent = () => {
+  return render(
+    <QueryClientProvider client={queryClient}>
+      <ManageOrganisationsDatagrid />
+    </QueryClientProvider>,
+  );
+};
+
+vi.mock('../DatagridCells', () => ({
+  OrganisationsIdCell: ({ org }: { org: string }) => <div>{org}</div>,
+  OrganisationsTypeCell: ({ org }: { org: string }) => <div>{org}</div>,
+  OrganisationsSurnameCell: ({ org }: { org: string }) => <div>{org}</div>,
+  OrganisationsEmailCell: ({ org }: { org: string }) => <div>{org}</div>,
+  OrganisationsPhoneCell: ({ org }: { org: string }) => <div>{org}</div>,
+  OrganisationsAddressCell: ({ org }: { org: string }) => <div>{org}</div>,
+  OrganisationsActionsCell: ({ org }: { org: string }) => <div>{org}</div>,
+}));
```

Comment:
I think you didn't need to mock all this, you could just render the app and navigate on the organisation list page and check the columns